requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy >= 1.17
+pydantic
+git+https://github.com/lsst/daf_butler@main#egg=lsst-daf-butler
+git+https://github.com/lsst/pex_config@main#egg=lsst-pex-config
+git+https://github.com/lsst/pipe_base@main#egg=lsst-pipe-base
+git+https://github.com/lsst/sphgeom@main#egg=lsst-sphgeom
+git+https://github.com/lsst/utils@main#egg=lsst-utils
```

Comment:
I don't have a clear idea on what packages need to be included here.

Reply:
I think we just can't complete the list right now, since it depends on a ton of things (e.g. `afw`) that aren't available on PyPI at all.  If we're using this file just for type-checking via the common workflow, we should add a comment to that effect, or if the workflow lets us, move all of the content here to `types.txt` and delete this file.

Reply:
It has to exist for the workflow to run at all but it can be empty.

Reply:
Even without `pydantic`? I'll leave this incomplete list on for now, since I've anyway put them in.